css/src/core/font-stack.scss

```diff
@@ -0,0 +1,25 @@
+html,
+body {
+	font-family: $normal-font-stack;
+	-webkit-font-smoothing: antialiased;
+	font-weight: $weight-normal;
+	text-rendering: optimizeLegibility;
```

Comment:
@wibjorn heads up for later (not needed for this PR)- turns out `optimizeLegibility` is often terrible for perf, at some point we should compare with `optimizeSpeed` and reserve `optimizeLegibility` for things like headings.

Reply:
Thanks for the heads up. Perhaps I can do some comparison now and see if there's a clear winner.

Reply:
@v-mingho can be a separate work item. Some articles on the subject:
https://tympanus.net/codrops/css_reference/text-rendering/
https://bocoup.com/blog/text-rendering

Reply:
New work item: https://dev.azure.com/ceapex/Engineering/_workitems/edit/397963